### ISSUE TITLE
SWR v0.4.1 crashes React Native when debugging

### ISSUE THREAD

**Issue by mikevercoelen** [vercel/swr#953]
When debugging React Native, SWR v0.4.1 is causing the following error:

`document is not defined`

It's due to all changes where we use document without checking if it exists: 

https://github.com/vercel/swr/compare/0.4.0...0.4.1

```
if (!IS_SERVER && window.addEventListener) {

// ...

window.addEventListener(
```

Became this:

```
if (!IS_SERVER && window.addEventListener && document.addEventListener) {

// ...

document.addEventListener(
```

V0.4.0 is working fine

**Comment by koba04** [comment]
I've created a PR to fix this. #963

**Comment by mikevercoelen** [comment]
Can we get a release for this please? As the latest version is broken on React Native, pretty essential to get this out there asap?

**Comment by koba04** [comment]
@mikevercoelen That makes sense for me.
@shuding Could you publish a new version?

**Comment by shuding** [comment]
Sorry for the delay, just published version 0.4.2 🙏
